iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java

```diff
@@ -292,9 +294,11 @@ private static Stream<RaiseIssue> provideRaiseIssue() {
   @Test
   void shouldParseTwoHelmFileInARowAndNotMixShiftedLocation() {
     final String originalSourceCode1 = K8_IDENTIFIERS + "{{ long helm code on line 5 }}\n{{ long helm code on line 6 }}";
-    final String transformedSourceCode1 = K8_IDENTIFIERS + "new_5_1: compliant #4\nnew_5_2: non_compliant #4\nnew_6_1: compliant #5\nnew_6_2: compliant #5";
+    final String transformedSourceCode1 = K8_IDENTIFIERS + "new_5_1: compliant #4\nnew_5_2: non_compliant #4\nnew_6_1: compliant " +
+      "#5\nnew_6_2: compliant #5";
     final String originalSourceCode2 = K8_IDENTIFIERS + "{{ helm code on line 5 }}\n{{ helm code on line 6 }}";
-    final String transformedSourceCode2 = K8_IDENTIFIERS + "new_5_1: compliant #4\nnew_5_2: compliant #4\nnew_6_1: compliant #5\nnew_6_2: non_compliant #5";
+    final String transformedSourceCode2 = K8_IDENTIFIERS + "new_5_1: compliant #4\nnew_5_2: compliant #4\nnew_6_1: compliant #5\nnew_6_2:" +
+      " non_compliant #5";
```

Comment:
Nitpick: I think a text block would be more readable here than splitting the string